recipes/pbbam/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+mkdir build
+cd build
+
+if [ `uname` == Darwin ]; then
+    export CXXFLAGS="${CXXFLAGS} -std=c++0x -stdlib=libc++"
+fi
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DPacBioBAM_build_docs=OFF -DPacBioBAM_build_tests=OFF
+make 
+cp lib/* $PREFIX/lib
```

Comment:
Is this really needed? `-DCMAKE_INSTALL_PREFIX=$PREFIX` should install everything in PREFIX